charts/sptribs-wa-post-deployment-ft-tests/Chart.yaml

```diff
@@ -8,5 +8,5 @@ maintainers:
   - name: HMCTS sptribs team
 dependencies:
   - name: java
-    version: 4.0.11
+    version: 5.2.1
     repository: '@hmctspublic'
```

Comment:
line 12 also looks outdated
Should look like this:
`
  - name: java
    version: 5.2.1
    repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
`